cmd/utils.go

```diff
@@ -2048,13 +2048,18 @@ func CheckValidSemver(version string) error {
 	return nil
 }
 
-func checkValidLicense(license string) error {
+func checkValidLicense(config *RootCommandConfig, license string) error {
```

Comment:
Instead of requiring the entire config, you only need `log *LoggingConfig` here.